flatpak-extension-runtime-fdo-openh264-watcher.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+After=systemd-tmpfiles-setup.service
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/systemctl restart flatpak-extension-runtime-fdo-openh264-ldconfig.service
```

Comment:
Instead of having a service that starts another service, is it possible to just have `flatpak-extension-runtime-fdo-openh264-ldconfig.path` that watches the that flatpak path?

Reply:
No, unless we only want it to be triggered once (e.g. not on every install/uninstall of the runtime) from what I checked - it will start the service but not restart it.

Reply:
You would have to drop the `RemainAfterExit=yes` from the service, though.

Reply:
Truth being told I got this pattern from https://superuser.com/questions/1171751/restart-systemd-service-automatically-whenever-a-directory-changes-any-file-ins and could make make it work reliably otherwise.

Reply:
I tried dropping RemainAfterExit (and even explicitly setting it to no) but could not make it work, but I tried so many different combinations that I may have let something slip.

Reply:
I think we can just go forward with this, but here's what I just tested:
test.service:
```
[Service]
Type=oneshot
ExecStart=/bin/true

[Install]
WantedBy=multi-user.target
```
test.path:
```
[Path]
PathChanged=/var/tmp/foo

[Install]
WantedBy=multi-user.target
```
The service is started at boot from multi-user.target, so if the runtime is already installed, `ldconfig` will be run. But if the path changes at any time after that then it will be run again.

Anyways, we can always fine tune this later if needed. What you have here should work fine.

Reply:
It may be the case I missed something but I'm pretty sure that was the first thing I tried and hit some issues (maybe because the dirs are symlinked and what not). As you said I would go with this and fine tune later if needed.